source/adapters/level_zero/event.cpp

```diff
@@ -742,7 +741,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urEventSetCallback(
   std::ignore = ExecStatus;
   std::ignore = Notify;
   std::ignore = UserData;
-  urPrint("[UR][L0] %s function not implemented!\n", __FUNCTION__);
+  logger::error("[UR][L0] {} function not implemented!\n", __FUNCTION__);
```

Comment:
Remove an extra newline.

Reply:
Done

Reply:
I believe `\n` is still here (I hope Patryk was talking about this, not something else)

Reply:
Yes, that is what I meant.

Reply:
This time for sure - Done